tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -16,10 +28,6 @@ ZTEST_BMEM volatile bool failed_in_isr;
 static ZTEST_BMEM const struct device *const uart_dev =
 	DEVICE_DT_GET(UART_NODE);
 
-static void read_abort_timeout(struct k_timer *timer);
-K_TIMER_DEFINE(read_abort_timer, read_abort_timeout, NULL);
```

Comment:
These were added in a previous commit. Probably should be reviewed by the commit author @nordic-krch

Reply:
Ok, the timer solution is a multi-threading disaster to be used in tests, which we should not do. Tests need to be deterministic.